dags/nct.py

```diff
@@ -1,12 +1,15 @@
 import datetime
+import airflow.operators.sensors
 from airflow.models import DAG
 import utils.helpers as helpers
+from operators.http_to_s3_transfer import HTTPToS3Transfer
 
 args = {
     'owner': 'airflow',
     'depends_on_past': False,
-    'start_date': datetime.datetime.utcnow(),
+    'start_date': datetime.datetime(2017, 1, 1),
```

Comment:
Yes. Airflow doesn't recommend a dynamic start date, because it would change every time the DAGs are loaded. Fixing it as part of this PR, but it's unrelated to the other changes.